jenkins/jobs/image-void.yaml

```diff
@@ -0,0 +1,92 @@
+- job:
+    name: "image-void"
```

Comment:
I think we want this to be image-voidlinux (and rename the .yaml files too) so we match the name of the old LXC template. It also looks like this is the generally used name on mirrors and other places for that distro.